scripts/giphy.py

```diff
@@ -0,0 +1,55 @@
+"""
+Respond to:
+
+    /giphy "phrase"
```

Comment:
I think it would be clearer to omit the quotes, because right now, it looks like it's required.